.github/CODEOWNERS

```diff
@@ -157,7 +157,7 @@
 # ServiceOwners:                                                   @rpsqrd @edyoung
 
 # PRLabel: %Batch
-/sdk/batch/                                                        @wiboris @dpwatrous
+/sdk/batch/                                                        @wiboris @dpwatrous @wanghoppe @skapur12
```

Comment:
- The account `wanghoppe` needs to make their Microsoft organization membership public.

- The account `dpwatrous` needs to make their Azure organization membership public.

Reply:
the accounts have been updated

Reply:
Confirmed!